src/o365/spfx/commands/project/project-rename.ts

```diff
@@ -90,81 +90,101 @@ class SpfxProjectRenameCommand extends BaseProjectCommand {
-          }
-        }
-
-        if (filePath.endsWith('config/package-solution.json')) {
-          if (updatedContent &&
-            updatedContent.solution &&
-            updatedContent.solution.name) {
-            updatedContent.solution.name = updatedContent.solution.name.replace(new RegExp(projectName, 'g'), args.options.newName);
-          }
-          if (updatedContent &&
-            updatedContent.solution &&
-            updatedContent.solution.id &&
-            args.options.generateNewId) {
-            updatedContent.solution.id = newId;
-          }
-          if (updatedContent &&
-            updatedContent.paths &&
-            updatedContent.paths.zippedPackage) {
-            updatedContent.paths.zippedPackage = updatedContent.paths.zippedPackage.replace(new RegExp(projectName, 'g'), args.options.newName);
-          }
-        }
-
-        if (filePath.endsWith('config/deploy-azure-storage.json')) {
-          if (updatedContent &&
-            updatedContent.container) {
-            updatedContent.container = args.options.newName;
-          }
-        }
-
-        fs.writeFileSync(filePath, JSON.stringify(updatedContent, null, 2), 'utf-8');
-      }
-      if (filePath.endsWith('README.md')) {
-        updatedContent = existingContent.replace(new RegExp(projectName, 'g'), args.options.newName);
-        fs.writeFileSync(filePath, updatedContent, 'utf-8');
+      const updatedContent = JSON.parse(existingContent);
+      if (updatedContent &&
+        updatedContent.solution &&
+        updatedContent.solution.name) {
+        updatedContent.solution.name = updatedContent.solution.name.replace(new RegExp(projectName, 'g'), args.options.newName);
       }
+      if (updatedContent &&
+        updatedContent.solution &&
+        updatedContent.solution.id &&
+        args.options.generateNewId) {
+        updatedContent.solution.id = newId;
+      }
+      if (updatedContent &&
+        updatedContent.paths &&
+        updatedContent.paths.zippedPackage) {
+        updatedContent.paths.zippedPackage = updatedContent.paths.zippedPackage.replace(new RegExp(projectName, 'g'), args.options.newName);
+      }
+      fs.writeFileSync(filePath, JSON.stringify(updatedContent, null, 2), 'utf-8');
+      if (this.debug) {
+        cmd.log(`Updated ${filePath.split('/').pop()}`);
+      }
+    }
+
+    const replaceDeployAzureStorageJsonContent = (filePath: string) => {
+      if (!fs.existsSync(filePath)) {
+        return;
+      }
+      const existingContent = fs.readFileSync(filePath, 'utf-8');
+      const updatedContent = JSON.parse(existingContent);
+      if (updatedContent &&
+        updatedContent.container) {
+        updatedContent.container = args.options.newName;
+      }
+      fs.writeFileSync(filePath, JSON.stringify(updatedContent, null, 2), 'utf-8');
+      if (this.debug) {
+        cmd.log(`Updated ${filePath.split('/').pop()}`);
```

Comment:
Rather than using fixed `/` we should use OS-specific path separator (`path.sep`). In this particular case, to get the file-/directory name from the path, it would be better to use `path.basename` (https://nodejs.org/api/path.html#path_path_basename_path_ext)